src/main.c

```diff
@@ -127,12 +127,16 @@ main (int argc, char *argv[])
 
   }
   while (next_option != -1);
+  
+  char *HOMEDIR = NULL;	 //const if there will be no changes to HOMEDIR
+  
+  HOMEDIR = getenv("HOME"); //getenv to get the HOME Directory 
+  if(HOMEDIR == NULL)
+    {printf("Error: Environmental variable $HOME can't be used. Unable to determine home directory");
```

Comment:
Thanks @ajphukan the code looks pretty good. There are some minor problems though that are common to "first-timers" contributing to other projects.

The brace should be on a line all by itself. I generally try to follow the coding standards used here: https://www.gnu.org/prep/standards/html_node/Formatting.html#Formatting

Also, you can look through other parts of the code to see how it's formatted.

Instead of printf, use fprintf (stderr, ...) (see other parts of the code to see how I use it).

Insert a \n at the end of the fprintf statement.